erpnext/patches/v13_0/set_per_picked_in_sales_order.py

```diff
@@ -0,0 +1,15 @@
+import frappe
+
+def execute():
+	frappe.reload_doctype("Sales Order")
```

Comment:
Use `reload_doc` instead with the `force` flag set to True. We've seen some issues with `reload_doctype`